woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-pre-orders.php

```diff
@@ -333,18 +338,25 @@ public function process_release_payment( $order ) {
 				if ( $this->get_gateway()->is_credit_card_gateway() ) {
 
 					$message = sprintf(
-							__( '%s %s Pre-Order Release Payment Approved: %s ending in %s (expires %s)', 'woocommerce-plugin-framework' ),
+							/* translators: Context: A payment is released for a pre-order. Placeholders: %1$s - Payment gateway name, %2$s - Either 'Authorization' or 'Charge' (untranslated), %3$s - Payment method type (e.g. 'PayPal', 'Credit Card', etc.), %4$s - Last four digits of the card or account, %5$s - Expiration date of the payment method */
```

Comment:
> Either 'Authorization' or 'Charge' (untranslated)

Is there a reason why these are untranslated? Perhaps something that was simply initially missed?

Reply:
I am not sure unfortunately, and I don't know if there's any legal requirement to keep them as such